app/services/orderService.js

```diff
@@ -26,29 +28,26 @@ class OrderService {
         const ticker = await exchange.fetchTicker(strategy.symbol);
         const price = ticker.ask;
 
-        const markets = await exchange.loadMarkets();
-        const market = markets[strategy.symbol];
-        const precision = market.market.precision.amount;
-        //TODO: make sure to reach the minimum amount and prize
+        //TODO: make sure to reach the minimum amount and prize by const markets = await exchange.loadMarkets()[strategy.symbol];
 
         let amount = 0;
         // Fetch the current price, calculate purchase amount this time by type
-        if (strategy.type === '1') {
-            amount = (strategy.base_limit / price).toFixed(precision);
+        if (strategy.type === STRATEGY_TYPE.NORMAL) {
+            amount = (strategy.base_limit / price).toFixed(6);
         } else {
-            amount = valueAveraging(strategy).toFixed(precision);
+            amount = valueAveraging(strategy).toFixed(6);
```

Comment:
In the older version, the precision was retrieved from the market, but here it is hardcoded as 6. Why was this value chosen?

Reply:
Retriving was my code, the original version was 6. I tried to make sure it fits market's requirement, but I found that this hardcode is applied in many functions of the old code, so I reverted to using 6.